systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -148,6 +149,27 @@ void testScramUserWithQuotas() {
         testUserWithQuotas(KafkaUserResource.scramShaUser(CLUSTER_NAME, "scramed-arnost").done());
     }
 
+    @Test
+    void testUserTemplate() {
+        String label = "test-label";
+        String annotation = "test-annotation";
+        KafkaUserResource.tlsUser(CLUSTER_NAME, USER_NAME)
+            .editSpec()
+                .editOrNewTemplate()
+                    .editOrNewSecret()
+                        .editOrNewMetadata()
+                            .addToLabels(label, label)
+                            .addToAnnotations(annotation, annotation)
```

Comment:
I think that in general it is better to use different keys and values - that way the test can make sure that the code doesn't not use the the other way around for example.